pandas/tests/io/json/test_pandas.py

```diff
@@ -1908,3 +1908,20 @@ def test_complex_data_tojson(self, data, expected):
         # GH41174
         result = data.to_json()
         assert result == expected
+
+    @pytest.mark.parametrize(
+        "data", [{"col1": [13342205958987758245, 12388075603347835679]}]
+    )
+    @pytest.mark.parametrize(
+        "expected",
+        [
+            '{"columns":["col1"],"index":[0,1],'
+            '"data":[[13342205958987758245],[12388075603347835679]]}'
+        ],
+    )
+    @pytest.mark.parametrize("orient", ["split"])
+    def test_json_uint64(self, data, expected, orient):
```

Comment:
This is only one test right? You don’t have to paeametrize over arguments that take only one value

Reply:
oh ok. Should I refactor and resubmit?

Reply:
You can just update this brancj